lib/util/index.js

```diff
@@ -1,22 +1,12 @@
 import * as distance from './distance'
 import * as itinerary from './itinerary'
-import * as map from './map'
-import * as profile from './profile'
-import * as query from './query'
-import * as reverse from './reverse'
 import * as state from './state'
-import * as time from './time'
 import * as ui from './ui'
 
 const OtpUtils = {
   distance,
   itinerary,
-  map,
-  profile,
-  query,
-  reverse,
   state,
-  time,
   ui
```

Comment:
Maybe we should export coreUtils in addition to whatever utils still live in otp-rr. I.e.,

```
const OtpUtils = {
  ...coreUtils,
  state
}
```

Reply:
I'm not sure... Should people import from OTP-ui directly? @evansiroky what do you think?

Reply:
Yea, you're probably right. We can keep as is for now.